### PR TITLE
[Vedanti]: Again Made new Changes in java project

### PR DESCRIPTION
**Pull request by kamde03Vedanti**
Hii..everyone , please check the code

### CODE REVIEW

**Comment by Shubham18Parakh** [comment]
your code is not appropriate need some changes once recheck.

File: src/RepositoriesTest/TC_002_Validate_Flipkart.java
Comment (pratikshakhote2000) [inline src/RepositoriesTest/TC_002_Validate_Flipkart.java:1]: HII vedanti.....code is not correct plz check and change as soon as possible

**Review by pratikshakhote2000 — APPROVED** [review]

File: src/RepositoriesTest/TC_002_Validate_Flipkart.java
Comment (Shubham18Parakh) [reply src/RepositoriesTest/TC_002_Validate_Flipkart.java:1]: done you can merge your code

**Review by Shubham18Parakh — APPROVED** [review]

**Review by Shubham18Parakh — APPROVED** [review]
no changes required

**Comment by pratikshakhote2000** [comment]
HII vedanti.....plz merge the code...all done

**Comment by pratikshakhote2000** [comment]
HII.....i appreciate ur work...ur doing well ...we will increase ur salary

**Pull request merged**